src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/PlaceholderTest.kt

```diff
@@ -0,0 +1,26 @@
+package fi.aalto.cs.apluscourses.e2e
+
+import fi.aalto.cs.apluscourses.e2e.fixtures.*
+import fi.aalto.cs.apluscourses.e2e.steps.CommonSteps
+import fi.aalto.cs.apluscourses.e2e.utils.StepLoggerInitializer
+import fi.aalto.cs.apluscourses.e2e.utils.uiTest
+import org.junit.Assert.*
+import org.junit.Test
+
+class PlaceholderTest {
+  init {
+    StepLoggerInitializer.init()
+  }
+
+  @Test
+  fun simpleTest() = uiTest {
```

Comment:
fails for me, but it's clear how to fix it (probably a matter of time outing or splitting properly)

Reply:
If you paste the console output, I can tweak timeouts.

Reply:
I will just fork and fix, no worries, but it's curious if it's the OS difference that makes it fail for me

Reply:
on the other hand, I see no problem if you fix it yourself 😛